notebooks/failure-type-classification/failure_type_functions.ipynb

```diff
@@ -84,7 +84,7 @@
     "            aws_secret_access_key=self.aws_secret_access_key,\n",
```

Comment:
<ul><li>Should we inherit (BaseEstimator, ClassifierMixin) in the FailureClassifier class so that we get the sklearn base classifier?</li><li>Do we need <code>data</code> to be a part of the init function? Since we are not using <code>self.data</code>  anywhere, we can just remove it.</li><li>For the heuristic functions, either we should have just the <code>data</code> parameter because  we can get grid from data or have <code>dates</code> and <code>grid</code>  as two parameters. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>